src/stories/3.Tx.stories.jsx

```diff
@@ -87,6 +100,134 @@ storiesOf('Tx/Fee Selector', module)
     />
   ))
 
+const TxTable = styled.table`
+  display: table;
+  border-collapse: collapse;
+  border-spacing: 0;
+  width: 100%;
+  margin: 16px 0;
+`
+
+const TxTableBody = styled.tbody`
+  display: table-row-group;
+  vertical-align: middle;
+  border-color: inherit;
+  tr:nth-child(odd) {
+    background-color: rgba(204, 198, 198, 0.3);
+  }
+`
+
+storiesOf('Tx/TransactionRow', module)
```

Comment:
All the stories render massive ETH incoming or outgoing numbers. I'd prefer to have some more realistic amounts to demo.